macOS/UnitTests/Menus/MoreOptionsMenuTests.swift

```diff
@@ -171,25 +171,26 @@ final class MoreOptionsMenuTests: XCTestCase {
         XCTAssertEqual(moreOptionsMenu.items[2].title, UserText.plusButtonNewTabMenuItem)
         XCTAssertEqual(moreOptionsMenu.items[3].title, UserText.newWindowMenuItem)
         XCTAssertEqual(moreOptionsMenu.items[4].title, UserText.newBurnerWindowMenuItem)
-        XCTAssertTrue(moreOptionsMenu.items[5].isSeparatorItem)
-        XCTAssertEqual(moreOptionsMenu.items[6].title, UserText.zoom)
-        XCTAssertTrue(moreOptionsMenu.items[7].isSeparatorItem)
-        XCTAssertEqual(moreOptionsMenu.items[8].title, UserText.bookmarks)
-        XCTAssertEqual(moreOptionsMenu.items[9].title, UserText.downloads)
-        XCTAssertEqual(moreOptionsMenu.items[10].title, UserText.passwordManagementTitle)
-        XCTAssertTrue(moreOptionsMenu.items[11].isSeparatorItem)
-        XCTAssertEqual(moreOptionsMenu.items[12].title, UserText.emailOptionsMenuItem)
-        XCTAssertTrue(moreOptionsMenu.items[13].isSeparatorItem)
-        XCTAssertEqual(moreOptionsMenu.items[14].title, UserText.subscriptionOptionsMenuItem)
-        XCTAssertFalse(moreOptionsMenu.items[14].hasSubmenu)
-        XCTAssertTrue(moreOptionsMenu.items[15].isSeparatorItem)
-        XCTAssertEqual(moreOptionsMenu.items[16].title, UserText.fireproofSite)
-        XCTAssertEqual(moreOptionsMenu.items[17].title, UserText.findInPageMenuItem)
-        XCTAssertEqual(moreOptionsMenu.items[18].title, UserText.shareMenuItem)
-        XCTAssertEqual(moreOptionsMenu.items[19].title, UserText.printMenuItem)
-        XCTAssertTrue(moreOptionsMenu.items[20].isSeparatorItem)
-        XCTAssertEqual(moreOptionsMenu.items[21].title, UserText.mainMenuHelp)
-        XCTAssertEqual(moreOptionsMenu.items[22].title, UserText.settings)
+        XCTAssertEqual(moreOptionsMenu.items[5].title, UserText.newAIChatMenuItem)
```

Comment:
The tests had the feature flag off for the duck.ai button. Now that I have removed the feature flag, it needs to be considered as well. This essentially offsets all indexes by +1 after the duck.ai button